src/index.js

```diff
@@ -2,13 +2,19 @@ import './style.css';
 
```

Comment:
- [ ] It would be nice if you fix your Live Demo link to make your readme professional 😊 If you don't have a live link or you didn't deploy your app so you can also remove this Live Demo section 👍 to make your readme descriptive and professional 😊